tests/form.spec.ts

```diff
@@ -0,0 +1,37 @@
+import { test, expect } from '@playwright/test'
+
+test.use({ headless: false })
+
+test('submit empty form', async ({ page }) => {
+
+    await page.goto('http://localhost:3000')
+    await page.click('button[type="submit"]')
+    await page.waitForSelector('[data-testid="Error"]');
+    const error = page.locator('[data-testid="Error"]');
+    await expect(error).toHaveText('Fill All the required fields'); 
+
```

Comment:
Remove unnecessary empty lines here.
```suggestion
    await page.goto('http://localhost:3000')
    await page.click('button[type="submit"]')
    await page.waitForSelector('[data-testid="Error"]');
    const error = page.locator('[data-testid="Error"]');
    await expect(error).toHaveText('Fill All the required fields'); 
```